packages/mobile/fastlane/metadata/android/es-419/full_description.txt

```diff
@@ -10,7 +10,7 @@ AHORRA Y GASTA VALOR ESTABLE
 Envíe, solicite y guarde Celo dólares, que son estables en relación con los dólares estadounidenses. Verifique fácilmente la propiedad de tu número de teléfono para comenzar. Después de eso, enviar o solicitar el pago a un amigo es fácil: todo lo que necesita es tu número de teléfono y una pequeña tarifa.
 
 SEGURO SEGURO
-Envíe de forma segura el valor a cualquier número de teléfono móvil o reciba un valor en tu número de teléfono. Para mantener tu valor seguro, proteja tu monedero con una clave de respaldo. Esta clave protege sus fondos para que solo usted pueda acceder a ellos, ¡no la pierda!
+Envíe de forma segura el valor a cualquier número de teléfono móvil o reciba un valor en tu número de teléfono. Para mantener tu valor seguro, proteja tu monedero con una clave de cuenta. Esta clave protege sus fondos para que solo usted pueda acceder a ellos, ¡no la pierda!
```

Comment:
Using _el/un/tu valor_ (which means _the/a/your value_) here feels very odd, but I am not sure what to replace it with and if it would create inconsistencies in other places

Reply:
ah-- agreed. I think this copy is actually unused 😅, so no trouble for now (and we really should delete soon)